addresses.go

```diff
@@ -94,3 +94,44 @@ func (*ListPublicIPAddresses) APIName() string {
 func (*ListPublicIPAddresses) response() interface{} {
 	return new(ListPublicIPAddressesResponse)
 }
+
+// ListRequest builds the ListAdresses request
+func (ip *IPAddress) ListRequest() (ListCommand, error) {
```

Comment:
rename it `ipaddress *IPAddress` (see codeclimate) and I'd move it next to the other `IPAddress` methods.

Reply:
👍